src/mlpack/tests/callback_test.cpp

```diff
@@ -0,0 +1,118 @@
+
+#include <ensmallen.hpp>
+#include <ensmallen_bits/callbacks/callbacks.hpp>
+#include <mlpack/core.hpp>
+#include <mlpack/methods/ann/ffn.hpp>
+#include <mlpack/methods/ann/rnn.hpp>
+#include <mlpack/methods/ann/loss_functions/mean_squared_error.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+using namespace mlpack;
+using namespace mlpack::ann;
+
+BOOST_AUTO_TEST_SUITE(CallbackTest);
+
+/**
+ * Test a FFN model with PrintLoss callback
```

Comment:
Missing stop (``.``) at the end, see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#comments for more informations.